src/main/resources/META-INF/reaktivity/amqp.idl

```diff
@@ -17,9 +17,57 @@ scope amqp
 {
     scope control
     {
+        struct AmqpRouteEx
+        {
+            string linkTargetAddress;
+            uint8 linkRole;
+        }
     }
 
     scope stream
     {
+        struct AmqpBeginEx
+        {
+            string containerId;
+            uint16 channelNumber;
```

Comment:
Suggest renaming `channelNumber` to `channel`.
In general, try to avoid putting type information into variable names.